CHANGELOG.md

```diff
@@ -0,0 +1,114 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.12.0]
+
+### Pull requests
```

Comment:
I think this will be hard in practice. Maybe we should use GH's RELEASES for that? It would automatically include the history.